src/components/ProfileDropdown.tsx

```diff
@@ -37,15 +37,16 @@ function ProfileDropdown({
             <div className="w-full border-border-dark dark:border-white border-b-[0.5px]">
               <div className="flex flex-row justify-between align-center gap-x-[20px] ">
                 <div className="flex flex-col w-full gap-[5px] cursor-pointer">
-                  <p className="font-semibold text-gray-600 dark:text-white px-4 py-2 hover:bg-gray-600 hover:text-gray-200 dark:hover:bg-gray-300 dark:hover:text-gray-900">
-                    <Link to="/dashboard/profile">{t('Profile')}</Link>
-                  </p>
-                  <p className="font-semibold text-gray-600 dark:text-white px-4 py-2 hover:bg-gray-600 hover:text-gray-200 dark:hover:bg-gray-300 dark:hover:text-gray-900">
-                    <Link to="#link">{t('Preference')}</Link>
-                  </p>
-                  <p className="font-semibold text-gray-600 dark:text-white px-4 py-2 hover:bg-gray-600 hover:text-gray-200 dark:hover:bg-gray-300 dark:hover:text-gray-900">
-                    <Link to="settings">{t('Privacy')}</Link>
-                  </p>
+                  <Link to="/dashboard/profile">
+                    <p className="font-semibold text-gray-600 dark:text-white px-4 py-2 hover:bg-gray-600 hover:text-gray-200 dark:hover:bg-gray-300 dark:hover:text-gray-900">
+                      {t('Profile')}
+                    </p>
+                  </Link>
+                  <Link to="settings">
+                    <p className="font-semibold text-gray-600 dark:text-white px-4 py-2 pb-4 hover:bg-gray-600 hover:text-gray-200 dark:hover:bg-gray-300 dark:hover:text-gray-900">
```

Comment:
There is no need for the paragraph as a children since the `Link` component also accepts `className` props and `text` as a children